src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -307,6 +329,7 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
                         min={1}
                         max={5}
                         defaultValue={this.props.computationPayload.election.firstDivisor}
+                        originalValue={this.props.settingsPayload.comparison.firstDivisor}
```

Comment:
Why was this done? Shouldn't display be controlled entirely by computation state?

Reply:
No, the original value is keeping track of the settings we are comparing against. Hence it is not tied to the computation payload.